packages/knip/src/plugins/convex/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://docs.convex.dev/home
+
+const title = 'Convex';
+
+const enablers = ['convex'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const entry: string[] = ['convex/*.config.@(js|jsx|ts|tsx)', 'convex/_generated/*.@(js|jsx|ts|tsx)'];
```

Comment:
Looking at the docs, there are no JSX config/generated files so seems to me we can remove every `x` here?

Reply:
Done!